reference_algorithms/threshold_baselines/README.md

```diff
@@ -0,0 +1,81 @@
+# Threshold Baselines
```

Comment:
Let's call them "Prize qualification baselines"
Try to just spell out what they are in prose and only use that 3 word name in paths and folders and filenames.

Reply:
We don't want people to confuse them with the qualification workloads/leaderboard, so it is good to also have the word "prize"

Reply:
Fixed